mysite/polls/views.py

```diff
@@ -322,8 +323,11 @@ def display_feed(request):
 
     return render(request, 'feed.html', {'images': all_images})
 
+
 def leaderboards(request):
     """A view to display the leaderboards"""
+    if not request.user.is_authenticated:
```

Comment:
Please have it redirect to the home page at /polls not to the login page

Reply:
Okay ill get that sorted for all the redirects when not authenticated.